### DISCUSSION TITLE
Guest role limited to SaaS version ?

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#6906] [General]
The changelog is talking about a new guest user role : https://www.getoutline.com/changelog/guest-role
After updating to v0.76.1 I couldn't see or activate it. Did you restrict that feature to the SaaS version ?

**Comment by tommoor** [comment]
Hey – the guest role is available on SaaS and licensed editions from v0.76.0 onwards.

**Reply by Limezy** [reply]
Thanks for your answer. That's a bit disappointing it's typically the kind of new features I'm waiting for and motivating my small github regular sponsorship. I hope you could consider one day a pricing option activating all features for someone like me which has only a few users, a need for slightly patching the app, and no need for enterprise SLA or support !

**Reply by tommoor** [reply]
While the sponsorship is of course appreciated – total sponsorship for Outline after almost 6 years of development comes to $300/mo. This is only enough to cover about 3 hours of development…

Of course I spend far more time than that on features for the community edition every month and pay contractors to work on the codebase also.
